src/app/components/navbar/navbar.component.spec.ts

```diff
@@ -53,4 +47,19 @@ describe('NavbarComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+it('should create the app --', async(() => {
+  let spy = spyOn(component, 'navbar');
+  window.dispatchEvent(new Event('scroll'));
+  expect(spy).toHaveBeenCalledTimes(0);
+}));
+
+
+it('should create the aeepp --', async(() => {
```

Comment:
Is this meant to be aeepp or app

Reply:
This was a reverse test for an if statement. I have changed the naming. Thanks.